download-singularity-img.nf

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env nextflow
+/*
+========================================================================================
+                         nf-core/rnafusion
+========================================================================================
+ nf-core/rnafusion Analysis Pipeline.
+ #### Homepage / Documentation
+ https://github.com/nf-core/rnafusion
+----------------------------------------------------------------------------------------
+*/
+
+nfcore_logo = """=======================================================
+                                          ,--./,-.
+          ___     __   __   __   ___     /,-._.--~\'
+    |\\ | |__  __ /  ` /  \\ |__) |__         }  {
+    | \\| |       \\__, \\__/ |  \\ |___     \\`-._,-`-,
+                                          `._,._,\'
+
+nf-core/rnafusion v${workflow.manifest.version}
+======================================================="""
+
+def helpMessage() {
+    nfcore_help = """
+    Usage:
+
+    The typical command for downloading singularity images is as follows:
+
+    nextflow run nf-core/rnafusion download-singularity-img.nf [OPTIONS] --outdir /path/to/output
```

Comment:
Shouldn't the command be `nextflow run nf-core/rnafusion/download-singularity-img.nf...` ?

Reply:
You are right! Fixed https://github.com/nf-core/rnafusion/commit/985d29a2aa5fe7b50ee408322d2a1d0ba9611918